tests/test_object_identifier.py

```diff
@@ -0,0 +1,117 @@
+import os
+import sys
+import unittest
+
+pkg_root = os.path.abspath(os.path.join(os.path.dirname(__file__), '..'))  # noqa
+sys.path.insert(0, pkg_root)  # noqa
+
+import dss
+from dss.storage.bundles import ObjectIdentifier, TombstoneID, FileFQID, BundleFQID, BUNDLE_PREFIX, FILE_PREFIX
+
+
+class TestObjectIdentifier(unittest.TestCase):
+
+    def test_to_str(self):
```

Comment:
For these kinds of tests I love using doctests but we don't have agreement on whether they are acceptable.

Reply:
Not familiar yet, I'll read up on that.